backends/sycl/ocloc_api.h

```diff
@@ -0,0 +1,103 @@
+//===------- ocloc_api.h --------------------------------------------------===//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// This file is copied from
+// https://github.com/intel/compute-runtime/blob/master/shared/offline_compiler/source/ocloc_api.h
```

Comment:
How does this square with our license?

Reply:
Apache-2.0 with LLVM-exception is functionally equivalent to Rust's MIT or Apache-2.0. MIT and BSD-2-Clause are also compatible so I think it's fine (we must preserve the statement).